src/SWP/Bundle/CoreBundle/MessageHandler/AnalyticsEventHandler.php

```diff
@@ -123,16 +93,9 @@ private function isHostMatchingTenant(string $host): bool
         return $host === $tenantHost;
     }
 
-    private function setTenant(Request $request): void
+    private function setTenant(string $httpReferrer): void
     {
-        $this->tenantContext->setTenant(
-            $this->tenantResolver->resolve(
-                $request->server->get('HTTP_REFERER',
-                    $request->query->get('host',
```

Comment:
this is used by 3d party events providers (like mobile app pageviews).

Reply:
it's there https://github.com/superdesk/web-publisher/pull/1041/files#diff-498b8ef8c7662ac5359ba7b5f6dcc861R44 in the new implementation

Reply:
👍